cli/src/cancel_invitation.rs

```diff
@@ -19,7 +19,7 @@ pub struct CancelInvitation {
     #[arg(short, long)]
     device: Option<String>,
     /// Invitation token
-    #[arg(short, long, value_parser = InvitationToken::from_hex)]
+    #[arg(short, long, value_parser = crate::clap_utils::token_from_hex::<InvitationToken>)]
```

Comment:
Implement a `InvitationToken::from_hex` instead of relying on the `FromHex` trait:
- this avoid the `token_from_hex` helper
- this also avoid having to expose `FromHex` trait in `libparsec_types::prelude`

Reply:
done